lib/graph.py

```diff
@@ -56,12 +56,12 @@ def __init__(self, fyle, line_width=40, **style):
         self.id = hash(fyle)
 
         path = pathlib.Path(fyle)
-        _, ext = os.path.splitext(path.name)
+        name, ext = os.path.splitext(path.name)
 
         self.style = style
 
         wrapper = get_text_wrapper(line_width)
-        path_name = "\n".join(wrapper.wrap(path.name))
+        path_name = "\n".join(wrapper.wrap(name))
```

Comment:
I'm confused as to why we wrap the path name without the extension? Wouldn't it make more sense to wrap it all together?

Reply:
I guess I did this because the wrapper prefers to break lines at spaces and punctuation including periods, and I didn't want it to put the extension on its own line. So we wrap first, and then the extension gets put on the same line as the rest of the file name